azure_pipelines/automation.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           -workspace MSAL.xcworkspace \
           -scheme "MSAL Test Automation (iOS)" \
           -sdk iphonesimulator \
-          -destination 'platform=iOS Simulator,name=iPhone 14,OS=17.2' \
+          -destination 'platform=iOS Simulator,name=iPhone 14,OS=16.4' \
```

Comment:
Any reason we cannot use latest iOS 17.x?

Reply:
was trying to resolve the automation pipeline failures for [MSAL Obj C Automations](https://identitydivision.visualstudio.com/IDDP/_build?definitionId=1284&_a=summary). there was an inconsistency in version numbers for what the pipeline was creating (MSAL Test Automation (iOS)_iphonesimulator16.4-x86_64.xctestrun) vs running (iOS 17.0)